src/webgpu/shader/execution/value_init.spec.ts

```diff
@@ -0,0 +1,370 @@
+export const description = `Test that variables in the shader are value initialized`;
+
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { GPUTest } from '../../gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+function paramTypeToTestType(ty: string): string {
```

Comment:
This name is confusing to me, becasue it looks like it is returning a value not a type. Should this be something like `paramValueForTestType` or `valueForTestType`?

Reply:
Or just use: `Type[ty].create(1).wgsl()`

Reply:
What does this do for `bool` map anything non-zero to `true`?